modules/apps/object/object-rest-impl/src/main/java/com/liferay/object/rest/internal/manager/v1_0/DefaultObjectEntryManagerImpl.java

```diff
@@ -140,16 +142,27 @@ public ObjectEntry addObjectEntry(
 
 		long groupId = getGroupId(objectDefinition, scopeKey);
 
-		return _toObjectEntry(
-			dtoConverterContext, objectDefinition,
+		Map<String, ObjectRelationship> objectRelationshipMap =
+			_getObjectRelationships(objectDefinition, objectEntry);
+
+		com.liferay.object.model.ObjectEntry newObjectEntry =
 			_objectEntryService.addObjectEntry(
 				groupId, objectDefinition.getObjectDefinitionId(),
 				_toObjectValues(
 					groupId, dtoConverterContext.getUserId(), objectDefinition,
 					objectEntry, 0L, dtoConverterContext.getLocale()),
 				_createServiceContext(
 					objectEntry.getProperties(),
-					dtoConverterContext.getUserId())));
+					dtoConverterContext.getUserId()));
+
+		if (GetterUtil.getBoolean(PropsUtil.get("feature.flag.LPS-153117"))) {
```

Comment:
For the following pull requests, I think it is a good idea to include all related to the feature flag in the same commit, just in case someone wants to check where was included just reviewing the git history